modules/gitops-release-notes-1-11-7.adoc

```diff
@@ -11,14 +11,16 @@
 == Errata updates
 
 [id="rhsa-2024-4972-gitops-1-11-7-security-update-advisory_{context}"]
-=== RHSA-2024:4972 - {gitops-title} 1.11.7 security update advisory
+=== RHSA-2024:4972 and RHBA-2024:6039 - {gitops-title} 1.11.7 security update advisory
 
 Issued: 2024-08-01
```

Comment:
@varshab1210 What about the Issue date? Should this updated to show the latest date (August 29)?

CC; @Dhruv-Soni11

Reply:
@Srivaralakshmi @Dhruv-Soni11 as I mentioned in slack, the version is v1.11.7-2.1724840231, not 1.11.7 itself. So we should be appending, not editing the existing section for 1.11.7